packages/@react-spectrum/breadcrumbs/stories/Breadcrumbs.stories.tsx

```diff
@@ -32,13 +33,25 @@ storiesOf('Breadcrumbs', module)
     'Default (with 7 items)',
     () => renderMany({})
   )
+  .add(
+    'multiline',
```

Comment:
Will rename stories to `hasHeading` if I receive confirmation that multiline is specifically for if the Breadcrumb's selected item is a heading (aka via <Heading> or isHeading prop)